### PR TITLE
chore: @gravity-ui/uikit -> 4.10.0

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/components/LabelWithPopover/LabelWithPopover.tsx [hunk 1]
```diff
@@ -3,14 +3,20 @@ import type {ReactNode} from 'react';
...
+export const LabelWithPopover = ({
+    text,
+    popoverContent,
+    className,
+    contentClassName,
+}: LabelWithPopoverProps) => (
     <div className={className}>
-        {text}
-        <HelpPopover content={popoverContent} />
+        {text + '\u00a0'}
```
Comment (artemmufazalov) [inline src/components/LabelWithPopover/LabelWithPopover.tsx:19]: Default offset was removed, added non-breaking space

File: src/components/TabletsStatistic/TabletsStatistic.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@
             background-color: var(--yc-color-base-positive);
         }
         &_state_yellow {
-            color: var(--yc-color-text-warning-medium);
+            color: var(--yc-color-text-warning);
```
Comment (artemmufazalov) [inline src/components/TabletsStatistic/TabletsStatistic.scss:27]: Color name was changed in uikit@4.0.0

File: src/utils/createToast.tsx [hunk 3]
```diff
@@ -10,13 +10,13 @@ interface CreateToastProps {
...
 function createToast({name, title, type, content}: CreateToastProps) {
-    return toaster.createToast({
+    return toaster.add({
         name: name ?? 'Request succeeded',
         title: title ?? 'Request succeeded',
         type: type ?? 'success',
         content: content,
         isClosable: true,
-        allowAutoHiding: type === 'success',
+        autoHiding: type === 'success' ? 5000 : false,
```
Comment (artemmufazalov) [inline src/utils/createToast.tsx:19]: 5000 was default before

File: package.json [hunk 4]
```diff
@@ -105,7 +105,7 @@
     "@gravity-ui/prettier-config": "^1.0.1",
     "@gravity-ui/stylelint-config": "^1.0.1",
     "@gravity-ui/tsconfig": "^1.0.0",
-    "@gravity-ui/uikit": "^3.20.2",
+    "@gravity-ui/uikit": "^4.10.0",
```
Comment (Raubzeug) [inline package.json:108]: It seems you should also update `navigation` package, because 0.3.1 version requires 3-rd uikit major.

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**